modular_zubbers/code/datums/mood_events/dominant_mood.dm

```diff
@@ -0,0 +1,20 @@
+/datum/mood_event/dominant/good_boy
+	description = span_nicegreen("I feel like a good THING!")
+	mood_change = 2
+	timeout = 3 MINUTES
+
+/datum/mood_event/dominant/need
+	description = span_warning("I need someone to make me a good THING!")
+	mood_change = -1
+	timeout = 3 MINUTES
+
+/datum/mood_event/dominant/add_effects(param)
+	. = ..()
+	var/mob/living/carbon/human/actual_owner = owner
+	var/good_x = "pet"
+	switch(actual_owner.gender)
+		if(MALE)
+			good_x = "boy"
+		if(FEMALE)
+			good_x = "girl"
+	description = replacetextEx(description, "THING", good_x)
```

Comment:
why, this is silly. somehow rework the verbiage or use one of the gender helpers like p_they or the likes. the code above alread gets the gender. What if the Gender isn't binary. Just return the helper.